cdk.json

```diff
@@ -15,18 +15,22 @@
     ]
   },
   "context": {
-    "env": "backup",
+    "account_name": "dev",
     "usernames": ["sandoval"],
```

Comment:
Should Laura's username be here still? What is this used for?

Reply:
I didn't want to touch this stuff in this PR, but I agree I don't think this is what we should do. I think we should add tags to users or something like that which can be added to roles instead.

Reply:
I can remove it later. I added it for permission to push to the ECR.